karma.conf.js

```diff
@@ -14,8 +14,8 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
       'bower_components/d3/d3.js',
```

Comment:
why is d3 loaded by bower while others deps are loaded via npm? Should be consistent if possible.